src/App.scss

```diff
@@ -2,18 +2,18 @@ iframe {
   display: none;
 }
 
-html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
-}
+// html {
+//   font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
+// }
 
-label,
-button {
-  cursor: pointer;
-}
+// label,
+// button {
+//   cursor: pointer;
+// }
 
-.error {
-  color: red;
-}
+// .error {
+//   color: red;
+// }
```

Comment:
why commented these styles?

Reply:
I tried to use Bulma for styling, so these styles were explicit but needed for tests, so I commented them out. However, it still works in both cases, so this time I included these styles.